src/lib/logger.ts

```diff
@@ -116,6 +120,13 @@ interface IDocumentInfo {
   changeCount?: number;
 }
 
+interface ITeacherNetworkInfo {
+  networkClassHash?: string;
+  networkClassName?: string;
+  networkUserId?: string;
+  networkUserName?: string;
+}
```

Comment:
I added this interface so we can optionally pass network specific info to logging functions like `logDocumentEvent` - I'm note sure if they names need tweaking - especially if they need to be unified with other instances of similar names.